src/ExpandableInput/ExpandableInput.js

```diff
@@ -213,10 +216,12 @@ module.exports = kind(
 	* @private
 	*/
 	inputSpotBlurred: function (inSender, inEvent) {
-		var eventType;
+		var eventType, lastEvent;
 		if (this.open && Spotlight.getPointerMode()) {
-			eventType = Spotlight.getLastEvent().type;
-			if (eventType !== 'onSpotlightFocus' && eventType !== 'mouseover') {
+			lastEvent = Spotlight.getLastEvent();
+			eventType = lastEvent.type;
+			if (eventType !== 'onSpotlightFocus' && eventType !== 'mouseover'
+				&& (!lastEvent.dispatchTarget || !lastEvent.dispatchTarget.isDescendantOf(this))) {
```

Comment:
Good catch - this is cruft from trying to handle an `ExpandableInput` edge case that went away by keeping track of an instance's history status, rather than trying to assume we can rely on all of the add/remove cases always pairing up.